docs/current_docs/api/snippets/services/bind-services/go/main.go

```diff
@@ -13,7 +13,7 @@ func (m *MyModule) HttpService() *dagger.Service {
 		From("python").
 		WithWorkdir("/srv").
 		WithNewFile("index.html", "Hello, world!").
-		WithExec([]string{"python", "-m", "http.server", "8080"}).
+		WithDefaultArgs([]string{"python", "-m", "http.server", "8080"}).
```

Comment:
If it's just for running the service I'd advise using the optional args in `AsService` rather than setting `WithDefaultArgs` and/or `WithEntrypoint`. These should be mostly used for affecting the metadata of a container image to be published.

Reply:
Done